src/actions/articles.js

```diff
@@ -82,3 +84,16 @@ export const editComment = (editedComment, id, articleSlug) => async (dispatch)
     toastr.error('Failed to edit comment.');
   }
 };
+export const editArticle = (articleSlug, articleData) => async (dispatch) => {
+  try {
+    const userToken = authUtils.getUserToken();
```

Comment:
There's no need adding the token to the request here again since we have set up interceptors on our axios instance